pages/api/cloudinary/[...media].tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  mediaHandlerConfig,
+  createMediaHandler,
+} from "next-tinacms-cloudinary/dist/handlers";
+
+// import { isAuthorized } from "@tinacms/auth";
+
+export const config = mediaHandlerConfig;
+// console.log("config:", config);
+
+export default createMediaHandler({
+  cloud_name: process.env.NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME,
+  api_key: process.env.NEXT_PUBLIC_CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_API_SECRET,
+  authorized: async (req, _res) => {
+    // console.log("req", req, process.env.NEXT_PUBLIC_USE_LOCAL_CLIENT);
+    try {
+      if (process.env.NEXT_PUBLIC_USE_LOCAL_CLIENT) return true;
+
+      return true;
```

Comment:
Can we clean this up? I'm not sure what this API route is supposed to be doing, or how the authorization fits into this.

Reply:
We've had to add this as a bandaid fix, as we've not got a clientId credential, and from the poor Tina CMS docs, we assume the documentation has changed in later versions (see current image implementation vs the one on Tina's Documentation - so we've written this to bypass _for now._

Spoke to @Stephen about this, it's not a perfect solution, but essentially it's because we need to update Tina/Next.js/ThemeUI etc but each one causes a knock-on with a bunch of caveats.